src/commands/buildCommands/configs/browserAppConfig.js

```diff
@@ -43,7 +42,8 @@ function injectHotReloadConfig(config) {
  * support translations (currently not supported by starter kit), but also
  * to determine the output path
  */
-function getConfig(localeCode) {
+function getConfig(localeCode, babel) {
+	const rules = require('./rules')(babel);
```

Comment:
consider keeping imports at the top level for consistency, maybe naming it something like `const getRules = require('./rules');`, and then calling it with `babel` here `const rules = getRules(babel);`